bootcamp-hub/values.yaml

```diff
@@ -51,6 +45,13 @@ jupyterhub:
       users:
         - earth-lab
         - jenp0277
+        - betatim
+        - lwasser
+    admin:
+      access: true
+      users:
+        - betatim
+        - lwasser
```

Comment:
Lines 50-54 setup the list of usernames who are admins. In this case betatim and lwasser. This method of setting who is an admin works independent of what authenticator (GitHub, Google, Hash, ...) we are using.